setup.py

```diff
@@ -23,13 +23,16 @@
         'symmetrical encryption of cached values using the python cryptography library'
     ),
     url='https://github.com/foundertherapy/django-redis-secure/',
-    download_url='https://github.com/foundertherapy/django-redis-secure/archive/1.0.0.tar.gz',
+    download_url='https://github.com/foundertherapy/django-redis-secure/archive/1.0.1.tar.gz',
     author='Ghaleb Khaled',
     author_email='ghaleb@foundertherapy.co',
     install_requires=[
-        'Django>=1.9.6',
+        'Django==1.9.6',
         'cryptography>=1.4',
         'django-redis==4.4.3',
+        'django-rq==0.9.1',
```

Comment:
well, I made this on purpose, since we actually "copied" some parts of `rq` & `django-rq` code and added in the project. So if we want to upgrade these libraries we need to update ours base code as well. 
@danaspiegel Thoughts?